src/javascript/DesignSystem/PickerDialog/PickerDialog.jsx

```diff
@@ -30,15 +30,15 @@ const styles = theme => ({
         padding: `0 ${theme.spacing.unit * 4}`,
         display: 'flex',
         alignItems: 'center',
-        justifyContent: 'space-between'
+        justifyContent: 'flex-end'
     },
     actionButtons: {
         '& button': {
             marginRight: theme.spacing.unit * 2
         }
     },
     actionUpload: {
-        display: 'flex',
+        display: 'none',
```

Comment:
If display none ? why not removing the element from the dom ?

Reply:
You can follow the slack discussion , PO said it will coming soon!

Reply:
Done